specification.markdown

```diff
@@ -301,7 +301,7 @@ Each `<element>` defines one `EBML Element` through the use of several attribute
 
 The `<element>` nodes contain a description of the meaning and use of the `EBML Element` stored within one or many `<documentation>` sub-elements.
 
-The `<element>` nodes MUST be arranged hierarchically according to the permitted structure of the EBML Document Type. An `<element>` node that defines an `EBML Element` which is a `Child Element` of another `Parent Element` MUST be stored as an immediate sub-element of the `<element>` node that defines the Parent Element. `<element>` nodes that define Level 0 Elements and Global Elements should be sub-elements of `<EBMLSchema>`.
+The `<element>` nodes MUST be arranged hierarchically according to the permitted structure of the EBML Document Type. An `<element>` node that defines an `EBML Element` which is a `Child Element` of another `Parent Element` MUST be stored as an immediate sub-element of the `<element>` node that defines the Parent Element. `<element>` nodes that define Root Level Elements and Global Elements should be sub-elements of `<EBMLSchema>`.
```

Comment:
With the intent of the PR, there's no longer a reason to say that `element nodes MUST be arranged hierarchically`. This paragraph can be cut down substantially just to:
> All `<element>` nodes MUST be sub-elements of `<EBMLSchema>`.

Since that is very brief I suggest just adding that sentence onto the prior single-sentence paragraph.